# Reference for `hub_sdk/helpers/exceptions.py`

!!! Note

    This file is available at [https://github.com/ultralytics/hub-sdk/blob/main/hub_sdk/helpers/exceptions.py](https://github.com/ultralytics/hub-sdk/blob/main/hub_sdk/helpers/exceptions.py). If you spot a problem please help fix it by [contributing](https://docs.ultralytics.com/help/contributing/) a [Pull Request](https://github.com/ultralytics/hub-sdk/edit/main/hub_sdk/helpers/exceptions.py) 🛠️. Thank you 🙏!

<br><br>

## ::: hub_sdk.helpers.exceptions.suppress_exceptions

<br><br>
